src/ui/components/tusktask/group/TaskCard.tsx

```
@@ -0,0 +1,165 @@
+                              You need to finish all sub-tasks before you can
+                              mark this task completed
+                            </span>
+                          </TooltipContent>
+                        </Tooltip>
+                      </TooltipProvider>
+                      <span>Sub-Tasks Overview</span>
+                    </h4>
+                    <div className="px-4 py-2 grid grid-cols-3 gap-2">
+                      <StatusOverviewCard
+                        title="Sub-Tasks"
+                        number={subTask.subTasks && subTask.subTasks.length}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)